src/apps/packet_filter/pcap_filter.lua

```diff
@@ -65,13 +67,15 @@ local basic_apps = require("apps.basic.basic_apps")
 function selftest ()
    print("selftest: pcap_filter")
    selftest_run(false, 3.726, 0.0009)
-   selftest_run(true,  3.800, 0.1)
+   selftest_run(true,  7.453, 0.001)
```

Comment:
So the new version accepts twice as many packets as the old version? Is one of them buggy and if so which one and why?